assets/i18n/language_en.properties

```diff
@@ -153,8 +153,10 @@ opinionComment.roomCleaning.sometimesCleanedAndGotCleanRoom = The room was not c
 opinionComment.roomCleaning.sometimesCleanedAndGotDirtyRoom = We received a dirty room, and it was not cleaned regularly.
 opinionComment.roomCleaning.alwaysDirty = The room was always dirty!
 
-opinionComment.roomBreaking.brokeButRepaired = There was a minor breakdown, but it was repaired.
-opinionComment.roomBreaking.brokeAndNotRepaired = There was a breakdown, and it was not repaired.
+opinionComment.roomBreaking.gotBrokenButRepaired = We received a room with a minor glitch, but it was repaired.
+opinionComment.roomBreaking.gotBrokenAndNotRepaired = We received a room with a minor glitch, and it was not repaired.
+opinionComment.roomBreaking.brokeButRepaired = There was a minor glitch, but it was repaired.
+opinionComment.roomBreaking.brokeAndNotRepaired = There was a minor glitch, and it was not repaired.
```

Comment:
glitch ? xD, jak już to malfunction albo defect i chyba bardziej coś w tym stylu by pasowało "We received a room with a small defect and it was not repaired."